src/cdk/text-field/_text-field.scss

```diff
@@ -8,11 +8,13 @@
   @keyframes cdk-text-field-autofill-end {/*!*/}
 
   .cdk-text-field-autofill-monitored:-webkit-autofill {
-    animation-name: cdk-text-field-autofill-start;
+    // In Chrome 80 we need a 1ms delay, or the animationstart event won't fire.
+    animation: cdk-text-field-autofill-start 0s 1ms;
```

Comment:
Should it be just `0` here and everywhere else?

Reply:
No, if I don't include `s` the `0` is interpreted as the `animation-iteration-count` and the `1ms` is interpreted as the `animation-duration`